main.py

```diff
@@ -104,7 +104,7 @@ def enroll_data(aliased=False):
                 updated_exp_year = data['exp_year']
                 updated.writerow([card_id, name, updated_number, updated_exp_month, updated_exp_year])
     print(
-        f'Successfully finish one-time enrollment process. '
+        f'Successfully finish bulk enrollment process. '
```

Comment:
```suggestion
        f'Successfully finished bulk enrollment process. '
```
Noticed a mistake here.